src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -15,6 +15,10 @@
 <head>
     <title>Profile</title>
     <link rel="stylesheet" href="/css/main.css">
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
```

Comment:
what are we using this for currently?

Reply:
I was using this to add some design to profile page.

Reply:
Right. I just mean where specifically? The submit button?

Reply:
In submit button, Description Field and this template has got different font as well. Originally, It was like
![new_one](https://user-images.githubusercontent.com/33577321/40929966-ecde8d82-67eb-11e8-863e-7cb123beb5fc.JPG)

Reply:
Looks nice!